microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1539,6 +1566,42 @@ The `CustomSchemeRegistration` allows registration of custom schemes in WebView2
 ---
 
 
+#### todo: add links to the above tab-set?  Nov 2023 incoming
+
+* For WinRT, options have been added to manage custom scheme registration when creating a `CoreWebView2Environment`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2CustomSchemeRegistration` Class:
+    * [CoreWebView2CustomSchemeRegistration.AllowedOrigins Property](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration.allowedorigins)
+    * [CoreWebView2CustomSchemeRegistration.SchemeName Property](/dotnet/api/microsoft.web.webview2.core.corewebview2customschemeregistration.schemename)
+
+* `CoreWebView2EnvironmentOptions` Class:
+   * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations)
+
+##### [WinRT/C#](#tab/winrtcsharp)
```

Comment:
for this one, only update WinRT tab to be the same as the .NET tab: i.e.

* [CoreWebView2CustomSchemeRegistration Class](microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration)
* `CoreWebView2EnvironmentOptions` Class:
    * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#customschemeregistrations)